stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -139,43 +132,46 @@ class FPDF:
         font_cache_dir: bool = ...,
     ) -> None: ...
     @property
-    def unifontsubset(self): ...
+    def unifontsubset(self) -> bool: ...
     @property
-    def epw(self): ...
+    def epw(self) -> float: ...
     @property
-    def eph(self): ...
+    def eph(self) -> float: ...
     def set_margin(self, margin: float) -> None: ...
     def set_margins(self, left: float, top: float, right: float = ...) -> None: ...
+    l_margin: float
     def set_left_margin(self, margin: float) -> None: ...
+    t_margin: float
     def set_top_margin(self, margin: float) -> None: ...
+    r_margin: float
     def set_right_margin(self, margin: float) -> None: ...
-    auto_page_break: Any
-    b_margin: Any
-    page_break_trigger: Any
+    auto_page_break: bool
+    b_margin: float
+    page_break_trigger: float
     def set_auto_page_break(self, auto: bool, margin: float = ...) -> None: ...
-    zoom_mode: Any
-    layout_mode: Any
-    def set_display_mode(self, zoom, layout: str = ...) -> None: ...
-    def set_compression(self, compress) -> None: ...
-    title: Any
+    zoom_mode: str | float
+    def set_display_mode(self, zoom: str | float, layout: str = ...) -> None: ...
+    compress: bool
+    def set_compression(self, compress: bool) -> None: ...
+    title: str
     def set_title(self, title: str) -> None: ...
-    lang: Any
+    lang: str
     def set_lang(self, lang: str) -> None: ...
-    subject: Any
+    subject: str
     def set_subject(self, subject: str) -> None: ...
-    author: Any
+    author: str
     def set_author(self, author: str) -> None: ...
-    keywords: Any
+    keywords: str
     def set_keywords(self, keywords: str) -> None: ...
-    creator: Any
+    creator: str
     def set_creator(self, creator: str) -> None: ...
-    producer: Any
+    producer: str
     def set_producer(self, producer: str) -> None: ...
-    creation_date: Any
+    creation_date: datetime.datetime | None
```

Comment:
Can also be `True`: https://github.com/PyFPDF/fpdf2/blob/48f837963b7ae1956a0e5b23815063d23c1964f5/fpdf/fpdf.py#L438